workflow/snakemake_rules/download.smk

```diff
@@ -5,8 +5,8 @@ rule download:
         metadata =  "data/{a_or_b}/metadata.tsv.gz",
         sequences = "data/{a_or_b}/sequences.fasta.xz"
     params:
-        metadata_url = "http://data.nextstrain.org/files/workflows/rsv/{a_or_b}/metadata.tsv.gz",
-        sequence_url = "http://data.nextstrain.org/files/workflows/rsv/{a_or_b}/sequences.fasta.xz"
+        metadata_url = "http://data.nextstrain.org/files/workflows/rsv/{a_or_b}/metadata_withRestricted.tsv.gz",
```

Comment:
I'm not a fine of camelCase URLs. I get caught easily with needing to think about casing.

Reply:
I am fine either way. change it.

Reply:
Resolved in https://github.com/nextstrain/rsv/pull/93/commits/9f9018f37f922df08da76ebaa92286f25924cb27.